20180125/MINUTES.md

```diff
@@ -0,0 +1,132 @@
+- Events problems: Tango Events no received at all. Due to rebooting VMs. Event system stops working. Rebooting the VM again, fix it.
+  Tango Forum: reported a list of possible problems related with Tango Events.
+
+__DESY__
+- In shutdown, so no feedback from users.
+- Planning the update of Sardana.
+
+__ALBA__
+- Moving a pseudomotor hangs a macro or MacroServer (randomly). Seems like the State event is not generated. Pseudomotor based on two physical motor. It will be debugged next Monday.
+  - Something similar saw at Solaris when the first movement of a pseudo was loosing the events (just after restart the pool). It will be reviewed by Gregorzs.
+  - Position of physical motors is sometimes not updated and that causes the pseudo position not being well calculated.
+- It would be useful to have a prepare-measurement (with number of repetitions) before the measurement, just once. (REPEATED from previous meeting)
+- It would be useful to have a call to cleanup-measurement at the end of the measurement, not at each step in step scan, just once. (Repeated)
+- Standard macros to manage the measurement groups
+- New repeat parameters syntax in sequencer (from text file)
+- New Macro API method that would be always called at the end of the macro execution, could be called "cleanup" or "restore"
+  - Can it be solved with a general hook? generalOnStop hook?
+
+__Actions__
+
+- [ ] Share macros for managing the measurement groups and make a standard macros for this.
+
+
+### 3. Collaborative Developments - Review
+
+__ALBA__
+- About positions on Continuous scans, it couldn't be started.
+
+__MAXIV__
+- We were more focused on updating Sardana and Taurus in the beamlines. Not much more done.
+
+__DESY__
+- PR created.
+
+__SOLARIS__
+- Working mostly on updates.
+
+__Common__
+
+- General Hooks pending from feedback from MAXIV and SOLARIS
+
+- Motor limits issue.
+  - Investigation done for accessing the limits of a motor.
+  - Avoid the use of attribute proxy and use the configuration of the attribute?
+- Taurusform compatibility with names. SOLARIS interested in working in this issue, but they are very busy with the upgrades.
+
+__Actions__
+- [ ] General hooks to be merged in the next Sardana release as an experimental API, then during the next months it will be evaluated and if no issues appear, then it will be official in the following release.
+- [ ] Integrate the actual solution for the Motor Limits Issue as an experimental solution and in the meantime try to find the best solution. Before the release.
+
+### 4. New Developments, Enhancements or Bug Fixes
+
+- Continue with the taurusform, logging, motor positions in continuous scans ([#643](https://github.com/sardana-org/sardana/issues/643)), and integrate the limit protection.
+
+### 5. Pending PR distribution and Jan18 release
+
+- General hooks
+- Pseudos limit protection
+- Sardana and Taurus4 URI
```

Comment:
* Here it is already Taurus4:)
* I think it is too optimistic to indlude it in the release. Well, actually I don't remember that we said that.